.github/workflows/ci_linux_x64_clang_debug.yml

```diff
@@ -56,9 +60,15 @@ jobs:
             -DCMAKE_BUILD_TYPE=Debug \
             -DIREE_BUILD_PYTHON_BINDINGS=ON \
             -DIREE_ENABLE_LLD=ON \
-            -DIREE_ENABLE_ASSERTIONS=ON
+            -DIREE_ENABLE_ASSERTIONS=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
+            -DCMAKE_C_COMPILER_LAUNCHER=sccache
       - name: CMake - build
-        run: cmake --build ${BUILD_DIR} -- -k 0
+        run: |
+          sccache --zero-stats
+          sccache --show-stats
+          cmake --build ${BUILD_DIR} -- -k 0
+          sccache --show-stats
```

Comment:
Low hit rate here too... hmm

https://github.com/iree-org/iree/actions/runs/10836779138/job/30071281859?pr=18489#step:6:7746
```
Compile requests                    5016
Compile requests executed           5016
Cache hits                           610
Cache hits (C/C++)                   610
Cache misses                        4406
Cache misses (C/C++)                4406
```
I dropped the cache from that workflow on `main` in https://github.com/iree-org/iree/pull/18255

Here are some logs before that commit that got 97% cache hit rates: https://github.com/iree-org/iree/actions/runs/10349302186/job/28643381760#step:4:11000
```
+ ccache --show-stats
Cacheable calls:   5837 / 5856 (99.68%)
  Hits:            5685 / 5837 (97.40%)
    Direct:         915 / 5685 (16.09%)
    Preprocessed:  4770 / 5685 (83.91%)
  Misses:           152 / 5837 ( 2.60%)
Uncacheable calls:   19 / 5856 ( 0.32%)
```

Reply:
Better on a previous run: https://github.com/iree-org/iree/actions/runs/10836229595/job/30069534660#step:6:8066